app/services/filter_service.rb

```diff
@@ -155,12 +155,12 @@ def build_custom_attr_query(query_hash, current_index)
     table_name = attribute_model == 'conversation_attribute' ? 'conversations' : 'contacts'
 
     query = if attribute_data_type == 'text'
-              "LOWER(#{table_name}.custom_attributes ->> '#{@attribute_key}')::#{attribute_data_type} #{filter_operator_value} #{query_operator} "
+              " ( LOWER(#{table_name}.custom_attributes ->> '#{@attribute_key}')::#{attribute_data_type} #{filter_operator_value} "
```

Comment:
Could you remove the extra spaces here?.

Reply:
done